tests/functional/interfaces-with-arguments.ts

```diff
@@ -0,0 +1,406 @@
+        @Field(type => Int, { nullable: true })
+        intValue2?: number;
+      }
+
+      @InterfaceType()
+      abstract class SampleInterface1 {
+        @Field(type => ID)
+        id: string;
+        @Field({ nullable: true })
+        interfaceStringField1?: string;
+
+        @Field()
+        interfaceFieldInlineArguments(
+          @Arg("intValue1") intValue1: number,
+          @Arg("intValue2") intValue2: number,
+        ): string {
+          throw new Error("Method not implemented.");
+        }
+        @Field()
+        interfaceFieldArgumentsType(@Args() args: SampleArgs1): string {
+          throw new Error("Method not implemented.");
+        }
+      }
+
+      @ObjectType({ implements: SampleInterface1 })
+      class SampleImplementingObject1 implements SampleInterface1 {
+        id: string;
+        interfaceStringField1?: string;
+
+        @Field()
+        url: string;
+
+        interfaceFieldInlineArguments(intValue1: number, intValue2: number): string {
+          return `${intValue1}-${intValue2}`;
+        }
+        interfaceFieldArgumentsType(args: SampleArgs1): string {
+          return `${args.intValue1}-${args.intValue2}`;
+        }
+
+        @Field()
+        implemetingObjectTypeFieldInlineArguments(
+          @Arg("intValue1") intValue1: number,
+          @Arg("intValue2") intValue2: number,
+        ): string {
+          return `${intValue1}-${intValue2}`;
+        }
+        @Field()
+        implemetingObjectTypeFieldArgumentsType(@Args() args: SampleArgs1): string {
+          return `${args.intValue1}-${args.intValue2}`;
+        }
+      }
+
+      @Resolver(of => SampleImplementingObject1)
+      class SampleResolver {
+        @Query(returns => [SampleImplementingObject1])
+        sampleQuery(): SampleImplementingObject1[] {
+          return [];
+        }
+
+        @FieldResolver(type => String)
+        sampleFieldResolver(@Root() sample: SampleImplementingObject1, @Args() args: SampleArgs1) {
+          return `${args.intValue1}-${args.intValue2}`;
+        }
+      }
+
+      // get builded schema info from retrospection
+      const schemaInfo = await getSchemaInfo({
+        resolvers: [SampleResolver],
+      });
+      schemaIntrospection = schemaInfo.schemaIntrospection;
+    });
+
+    it("should generate schema without errors", async () => {
+      expect(schemaIntrospection).toBeDefined();
+    });
+
+    it("should have proper arguments for the interface field with inline arguments", async () => {
+      const sampleField = (schemaIntrospection.types.find(
+        type => type.name === "SampleInterface1",
+      ) as IntrospectionInterfaceType).fields.find(
+        f => f.name === "interfaceFieldInlineArguments",
+      ) as IntrospectionField;
+
+      expect(sampleField.args).toBeDefined();
+      expect(sampleField.args.length).toEqual(2);
+      expect(
+        sampleField.args.every(arg => ["intValue1", "intValue2"].includes(arg.name)),
```

Comment:
Less magic, more explicitness - please use `sampleField.args.find` to get two variables and normal assertion, just like in `resolvers` test suite, instead of `every`, `includes` and `toBeTruthy`.